ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/StandaloneExecutorTest.java

```diff
@@ -16,76 +16,229 @@
+import io.confluent.ksql.parser.tree.Query;
+import io.confluent.ksql.parser.tree.SetProperty;
+import io.confluent.ksql.parser.tree.UnsetProperty;
+import io.confluent.ksql.serde.DataSource.DataSourceType;
 import io.confluent.ksql.util.KsqlConfig;
+import io.confluent.ksql.util.KsqlException;
+import io.confluent.ksql.util.Pair;
 import io.confluent.ksql.util.PersistentQueryMetadata;
 import io.confluent.ksql.util.QueryMetadata;
-import java.io.FileOutputStream;
 import java.io.IOException;
-import java.nio.charset.StandardCharsets;
 import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
 import org.apache.kafka.test.TestUtils;
 import org.easymock.EasyMock;
 import org.junit.Before;
 import org.junit.Test;
 
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
 public class StandaloneExecutorTest {
 
   private final KsqlConfig ksqlConfig = new KsqlConfig(Collections.emptyMap());
   private final KsqlEngine engine = EasyMock.niceMock(KsqlEngine.class);
   private final UdfLoader udfLoader = EasyMock.niceMock(UdfLoader.class);
-  private final String query = "select * from bar;";
-  private StandaloneExecutor executor;
+  private String queriesFile;
+  private StandaloneExecutor standaloneExecutor;
+  private MetaStore metaStore;
+  private final Map<String, Object> props = ImmutableMap.of();
+  private final QualifiedName qualifiedName = QualifiedName.of("Test");
+
+  final Query query = EasyMock.niceMock(Query.class);
+  final QueryMetadata persistentQueryMetadata = EasyMock.niceMock(PersistentQueryMetadata.class);
 
   @Before
   public void before() throws IOException {
-    final String queriesFile = TestUtils.tempFile().getPath();
-    executor = new StandaloneExecutor(ksqlConfig, engine, queriesFile, udfLoader);
-    try(final FileOutputStream out = new FileOutputStream(queriesFile)) {
-      out.write(query.getBytes(StandardCharsets.UTF_8));
+    queriesFile = TestUtils.tempFile().getPath();
+    standaloneExecutor =
+        new StandaloneExecutor(ksqlConfig, engine, queriesFile, udfLoader);
+    metaStore = EasyMock.niceMock(MetaStore.class);
+    EasyMock.expect(engine.getMetaStore()).andReturn(metaStore);
+  }
+
+
+  @Test
+  public void shouldFailDropStatement() throws IOException {
+
+    EasyMock.expect(engine.parseStatements(anyString(), anyObject()))
+        .andReturn(ImmutableList.of(new Pair<>("CS",
+            new CreateStream(qualifiedName, Collections.emptyList(), false, Collections.emptyMap())),
+            new Pair<>("DROP",
+                new DropStream(qualifiedName, false, false))));
+
+    EasyMock.expect(engine.buildMultipleQueries("CS", ksqlConfig, props))
+        .andReturn(Collections.emptyList());
+
+
+    EasyMock.replay(engine);
+    try {
+      standaloneExecutor.start();
+    } catch (KsqlException e) {
+      assertThat(e.getMessage(), equalTo("Ignoring statements: DROP\n"
```

Comment:
Prefer `ExpectedException` class in favour of try/catch in your tests.

Reply:
Replaced it with `ExpectedException`.